packages.json

```diff
@@ -42,7 +38,6 @@
                 "solaar-udev",
                 "symlinks",
                 "tcpdump",
-                "tmux",
```

Comment:
This was added nearly a year ago ( #190 ) due to multiple contributors/users who believed it to be a useful, lightweight tool which is valuable on the host main images and who realized it was being added on several downstream images.

I'm not inclined to remove it.

Reply:
Second this. Similar to vim this is a popular tool for folks and adds almost no size to the image to include it.